### ISSUE TITLE
obj export is not centered at 0,0,0

### ISSUE THREAD

**Issue by EstevanBR** [ephtracy/ephtracy.github.io#156]
GIVEN: a .vox project, at 16x16x16 res, basically a cube
WHEN: exporting to .obj
THEN: the origin of the mesh should be in the center of the bounding box, and be centered on 0,0

ACTUAL: the mesh is not centered, the origin is at 0,0,0, but the mesh itself is not centered properly.
![image](https://user-images.githubusercontent.com/5488655/82169580-c06aa980-9876-11ea-81ac-a7f1a4350919.png)

Using 99.5, and blender 2.8

**Comment by LanceJZ** [comment]
Yeah, I'm having the same issue. I use it to make game assets for Unity, and I have to center them. This has been a thing on the last one too. The pivot setting is gone from the config file.

**Comment by Sandstedt** [comment]
Does it work if you move the object in the world view before exporting? (just as a temp workaround)

**Comment by ephtracy** [comment]
In the latest version, you can select "Export->Local Space" in the IO panel.
